docs/advanced/hgq.rst

```diff
@@ -0,0 +1,49 @@
+==============
+High Granularity Quantization (HGQ)
+==============
+
+.. image:: https://github.com/calad0i/HGQ/actions/workflows/sphinx-build.yml/badge.svg
+   :target: https://calad0i.github.io/HGQ/
+.. image:: https://badge.fury.io/py/hgq.svg
+   :target: https://badge.fury.io/py/hgq
+.. image:: https://img.shields.io/badge/arXiv-2405.00645-b31b1b.svg
+   :target: https://arxiv.org/abs/2405.00645
+
+High Granularity Quantization (HGQ) is a gradient-based automatic bitwidth optimization and quantization-aware training algorithm for neural networks to be deployed on FPGAs, By laveraging gradients, it allows for bitwidth optimization at arbitrary granularity, up to per-weight and per-activation level.
+
+.. image:: https://calad0i.github.io/HGQ/_images/overview.svg
+   :alt: Overview of HGQ
+   :align: center
+
+Conversion of models from High Granularity Quantization (HGQ) is fully supported. The HGQ models are first converted to proxy model format, which can then be parsed by hls4ml bit-accurately. Below is an example of how to create a HGQ model and converting it to hls4ml.
```

Comment:
You already introduced the HGQ abbreviation, no need to write it as `High Granularity Quantization (HGQ)` again. Also, it would be more clear if you refered to HGQ as a library in the first sentense. Like, "Conversion of models made with HGQ library is fully supported".

In the last sentence, pick a style, either "example of how to create ... and convert..." or "example of creating ... and converting ..."